ls1intum/Artemis
# `Quiz exercises`: Improve user experience for the re-evaluation page

### Checklist
#### General


#### Client

### Motivation and Context



Buttons and inputs are not consistent on the quiz re-evaluation page. Furthermore, interaction with some inputs is difficult. The page is also not mobile-friendly.

https://user-images.githubusercontent.com/10885503/207867627-334127b5-142c-4016-bb36-9f63a90c15da.mov

### Description


### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course

1. Log in to Artemis
2. Navigate to Course Administration
3. Select the Course
4. Create Quiz Exercise
5. Add 1 Multiple Choice Option
6. Set duration to 1 second
7. Click Save
8. Click Start Exercise
9. Wait for 6 seconds and refresh the page
10. Click Re-evaluate
11. Verify that the reevaluation page can be used and works.

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage





### Screenshots



[Before Improvement (Desktop)](https://user-images.githubusercontent.com/10885503/207866683-8d5738cc-741d-4973-8728-9089a0b7dcae.png)

[Before Improvement (Mobile)](https://user-images.githubusercontent.com/10885503/207867082-efd895c2-990a-4d7a-bf1c-7a9ac695eac6.png)

[After Improvement (Desktop)](https://user-images.githubusercontent.com/10885503/207865502-bd5f9f9e-31f9-422f-ac48-178dbd8b53d1.png)

[After Improvement (Mobile)](https://user-images.githubusercontent.com/10885503/207866942-798b48f7-e5ba-4d03-9ba2-a7fe43d17ad1.png)
